cmd/certificates/sign.go

```diff
@@ -5,59 +5,57 @@ package main
 
 import (
 	"crypto/x509"
+	"fmt"
+	"log"
 
 	"github.com/spf13/cobra"
 	"github.com/zeebo/errs"
 
-	"storj.io/storj/pkg/cfgstruct"
 	"storj.io/storj/pkg/identity"
 )
 
 var (
 	signCmd = &cobra.Command{
-		Use:   "sign",
+		Use:   "sign [signee identity-dir]",
 		Short: "Sign a CA and update corresponding CA and identity certificate chains",
+		Args:  cobra.ExactArgs(1),
 		RunE:  cmdSign,
 	}
 
 	signCfg struct {
-		CA       identity.FullCAConfig
-		Identity identity.Config
+		SigneeCACfg    identity.PeerCAConfig
+		SigneeIdentCfg identity.PeerConfig
 		// NB: defaults to same as CA
 		Signer identity.FullCAConfig
 	}
 )
 
-func init() {
-	rootCmd.AddCommand(signCmd)
-	cfgstruct.Bind(signCmd.Flags(), &signCfg, cfgstruct.ConfDir(defaultConfDir))
-}
-
 func cmdSign(cmd *cobra.Command, args []string) error {
-	ca, err := signCfg.CA.Load()
+	ca, err := signCfg.SigneeCACfg.Load()
 	if err != nil {
 		return err
 	}
 
-	ident, err := signCfg.Identity.Load()
+	ident, err := signCfg.SigneeIdentCfg.Load()
 	if err != nil {
-		return err
+		log.Printf("unable to load identity: %s\n", err.Error())
```

Comment:
similar question - why are we logging the error here and not on line 36?

Reply:
(same)